src/main/java/Interfaces/IHardwareMaintenance.java

```diff
@@ -0,0 +1,55 @@
+package Interfaces;
+
+import Issues.Task;
+
+public class IHardwareMaintenance extends Task implements IHardware {
```

Comment:
only interfaces should have `I` in the name
```suggestion
public class HardwareMaintenance extends Task implements IHardware {
```

Reply:
also please rename all Task-related classes using this pattern: HardwareMaintenanceTask